### PR TITLE
[dev-v2.5] Update calico + canal to v3.16.5 in v1.19.4-rancher1-2

### PR DESCRIPTION
**Pull request by pennyscissors**
Updates calico's and canal's templates/tags to address the following issue: https://github.com/rancher/rancher/issues/30324
Updates `v1.19.4-rancher1-2` as it's the only template using calico images with tag  v3.16.1

### CODE REVIEW

**Comment by luthermonson** [comment]
PR is good but need it all squashed to one commit. ill approve if you want to do that with the gh ui or force push and ping me and ill reapprove.

**Review by luthermonson — DISMISSED** [review]

**Comment by PennyScissors** [comment]
@luthermonson No problem, I'll squash before merging.

**Review by kinarashah — APPROVED** [review]

**Pull request merged**
